src/Microsoft.AspNet.Razor.Runtime/TagHelpers/HtmlTargetElementAttribute.cs

```diff
@@ -79,5 +79,11 @@ public HtmlTargetElementAttribute(string tag)
         /// </para>
         /// </remarks>
         public TagStructure TagStructure { get; set; }
+
+        /// <summary>
+        /// The required HTML element name of the direct parent.
+        /// </summary>
+        /// <remarks>A <c>null</c> value indicates any HTML element name is appropriate.</remarks>
```

Comment:
Not sure what you mean. Like invalid characters? If so we don't mention such restrictions for `AllowedChildren` or `TagName`